src/client/components/SliderComponent/SliderComponent.js

```diff
@@ -50,7 +50,10 @@ class Slider extends React.Component {
     return (
       <div className="slideshow_container">
         <div className="arrow_component">
-          <ArrowsComponent clickLeftArrow={this.gotoPreviousSlide} clickRightArrow={this.gotoNextSlide} />
+          <ArrowsComponent
+            clickLeftArrow={this.gotoPreviousSlide}
+            clickRightArrow={this.gotoNextSlide}
+          />
```

Comment:
This looks good but after the merging both arrows, they are both on the left side of the screen. You need to separate them to locate the image in the middle.

Reply:
Thanks for pointing it out. I will do the necessary changes

Reply:
Implemented the changes